packages/blocks/src/embed-github-block/embed-github-block.ts

```diff
@@ -1,44 +1,332 @@
+    if (createdAt) {
+      const date = new Date(createdAt);
+      dateText = date.toLocaleDateString('en-US', {
+        month: 'short',
+        day: 'numeric',
+      });
+      const day = date.getDate();
+      const suffix =
+        ['th', 'st', 'nd', 'rd'][((day / 10) | 0) !== 1 ? day % 10 : 4] || 'th';
+      dateText = dateText.replace(/\d+/, `${day}${suffix}`);
+    }
+
+    this.cardStyle = style;
+
+    return this.renderEmbed(
+      () => html`
+        <div
+          style=${styleMap({
+            position: 'relative',
+          })}
+        >
+          <div
+            ${this.isInSurface ? nothing : ref(this._whenHover.setReference)}
+            class=${classMap({
+              'affine-embed-github-block': true,
+              loading,
+              [style]: true,
+            })}
+            @click=${this._handleClick}
+            @dblclick=${this._handleDoubleClick}
+          >
+            <div class="affine-embed-github-content">
+              <div class="affine-embed-github-content-title">
+                <div class="affine-embed-github-content-title-icons">
+                  <div class="affine-embed-github-content-title-site-icon">
+                    ${titleIcon}
+                  </div>
+
+                  ${status && statusText
+                    ? html`<div
+                        class=${classMap({
+                          'affine-embed-github-content-title-status-icon': true,
+                          [githubType]: true,
+                          [status]: true,
+                          success: statusReason === 'completed',
+                          failure: statusReason === 'not_planned',
+                        })}
+                      >
+                        ${statusIcon}
+
+                        <span>${statusText}</span>
+                      </div>`
+                    : nothing}
+                </div>
+
+                <div class="affine-embed-github-content-title-text">
+                  ${titleText}
+                </div>
+              </div>
+
+              <div class="affine-embed-github-content-description">
+                ${descriptionText}
+              </div>
+
+              ${githubType === 'issue' && assignees
+                ? html`
+                    <div class="affine-embed-github-content-assignees">
+                      <div
+                        class="affine-embed-github-content-assignees-text label"
+                      >
+                        Assignees
+                      </div>
+
+                      <div
+                        class="affine-embed-github-content-assignees-text users"
+                      >
+                        ${assignees.length === 0
+                          ? html`<span
+                              class="affine-embed-github-content-assignees-text-users placeholder"
+                              >No one</span
+                            >`
+                          : assignees.map(
```

Comment:
We should render lists by the `repeat` directive in lit to make sure each item has a key,

Reply:
@Saul-Mirone corrected